### ISSUE TITLE
World can't be stored in var

### ISSUE THREAD

**Issue by uiytt** [SkriptLang/Skript#2059]
### Description
World are not stored in variables

### Steps to Reproduce
DOESN'T WORK
`command /test:
	trigger:
		set {_world} to player's world
		set {_world}'s time to 6:00`
WORKING : 
`command /test:
	trigger:
		set {_world} to player's world
		set world("%{_world}%")'s time to 6:00`
### Expected Behavior
The first one need to set time in the player's world to 6:00 (0 ticks)

### Server Information
* **Server version/platform:**     spigot-1.12.2
* **Skript version:**                2.3.6

**Comment by TheBentoBox** [comment]
Can anyone recreate this on the latest version?

**Comment by HAHAYOUDEAD** [comment]
Your syntax doesn't work, but this one does:

```
command /test1:
    trigger:
        set {_world} to player's world
        set time in {_world} to 6:00
```

You can also do `broadcast "%{_world}%"` to check if variable is set or not (it is, meaning your title is incorrect)

tested on paper-59 (1.14.2), Skript 2.4-alpha2

**Comment by uiytt** [comment]
In 2.3.6 : 
broadcast "%{_world}%" return <none> 

But i guess it was corrected in 2.4

**Comment by TheBentoBox** [comment]
I also cannot reproduce in 1.14.
